style/index.html

```diff
@@ -117,12 +117,11 @@ <h3 class="player-font player3-total-text">Total Score: 0</h3>
   </aside>  
 </body>
 
-  <script type="text/javascript" src="javascript/domUpdates.js"></script>
-  <script type="text/javascript" src="javascript/data.js"></script>
-  <script type="text/javascript" src="javascript/Player.js"></script>
-  <script type="text/javascript" src="javascript/Game.js"></script>
-  <script type="text/javascript" src="javascript/Round.js"></script>
-  <script type="text/javascript" src="javascript/Puzzle.js"></script>
-  <script type="text/javascript" src="javascript/Wheel.js"></script>
-  <script type="text/javascript" src="javascript/index.js"></script>
+  <script type="text/javascript" src="../lib/domUpdates.js"></script>
+  <script type="text/javascript" src="../lib/data.js"></script>
+  <script type="text/javascript" src="../lib/Player.js"></script>
+  <script type="text/javascript" src="../lib/Game.js"></script>
+  <script type="text/javascript" src="../lib/Puzzle.js"></script>
+  <script type="text/javascript" src="../lib/Wheel.js"></script>
+  <script type="text/javascript" src="../lib/scripts.js"></script>
```

Comment:
Im assuming we're changing the name of the previous 'index.js' to 'scripts.js' to clear some confusion?